libraries/CAN/test/unit/Unity/unity.c

```diff
@@ -0,0 +1,2466 @@
+/* =========================================================================
+    Unity Project - A Test Framework for C
+    Copyright (c) 2007-21 Mike Karlesky, Mark VanderVoord, Greg Williams
+    [Released under MIT License. Please refer to license.txt for details]
+============================================================================ */
+
+#include "unity.h"
```

Comment:
Remove unity sources.

Reply:
Provide minimum info about the version used, and how to install it instead?

Reply:
I think I need to talk with Jens about the newest Unity setup. Before that I can remove them...

Reply:
yep, but a couple of lines on how to tests these? or there is more to it?
cd _<directory>_
git clone _www.blabla.unity.git_
make _test_

Reply:
This unity setup I'm currently using only picks some of the unity source code, along with a few modifications. So I guess it's a similar problem with XMCLib. I will discuss it with Jens 😄 He told me earlier that he had a new setup.

Reply:
mmm okay, if there are modifications it is might not so convenient to just remove it :) Okay, let´s see the new setup.